### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by KiwiColoredHorror**

### CODE REVIEW

File: tutorials/matrices_and_transforms.rst [hunk 1]
```diff
@@ -239,15 +240,15 @@ the scale). It will leave the origin alone:
 .. image:: /img/tutomat15.png
 
 These kind of operations in matrices are accumulative. It means every
-one starts relative to the previous one. For those that have been living
+one starts relative to the previous one. For those which have been living
```
Comment (bojidar-bg) [inline tutorials/matrices_and_transforms.rst]: Maybe `who` would be even better?

File: tutorials/matrices_and_transforms.rst [hunk 2]
```diff
@@ -239,15 +240,15 @@ the scale). It will leave the origin alone:
...
 on this planet long enough, a good reference of how transform works is
 this:
 
 .. image:: /img/tutomat16.png
 
 A matrix is used similarly to a turtle. The turtle most likely had a
-matrix inside (and you are likely learning this may years *after*
-discovering Santa is not real).
+matrix inside (and you are likely learning this many years *after*
+discovering that Santa is not real).
```
Comment (bojidar-bg) [inline tutorials/matrices_and_transforms.rst]: `that` isn't essential in this context IMO.

File: tutorials/matrices_and_transforms.rst (hunk 1 above)
Comment (KiwiColoredHorror) [inline tutorials/matrices_and_transforms.rst]: Yes, it is better.

File: tutorials/matrices_and_transforms.rst (hunk 2 above)
Comment (KiwiColoredHorror) [inline tutorials/matrices_and_transforms.rst]: Both cases are correct (I just checked them) so I'll remove it. Thanks

**Pull request merged**